server/src/main/java/org/gluu/oxtrust/action/PasswordValidationAction.java

```diff
@@ -74,18 +74,30 @@ public void storeNewPassword(GluuCustomPerson person, boolean validateOldPasswor
 
 			if (!resultValidateOldPassword) {
 				if (graphValidator == null) {
-					FacesContext.getCurrentInstance().addMessage(null,
-							new FacesMessage(FacesMessage.SEVERITY_ERROR, "Old password isn't valid!", "Old password isn't valid!"));
-					
+					FacesContext.getCurrentInstance().addMessage(null, new FacesMessage(FacesMessage.SEVERITY_ERROR,
```

Comment:
You not changed these lines but you did reformat... Anyway we need to replace:
```
FacesContext.getCurrentInstance().addMessage(null, new FacesMessage(FacesMessage.SEVERITY_ERROR....
```
with:
```
	@Inject
	private FacesMessages facesMessages;

...
		facesMessages.add(FacesMessage.SEVERITY_ERROR, ".....");

```

This will make code simplier